db/models.py

```diff
@@ -1 +1,50 @@
 from django.db import models
+
+
+class Race(models.Model):
+    names_choices = [
```

Comment:
NAMES_CHOICES should be capitalized it is a convention to signify that this sequence is a constant and to differentiate from regular fields. Also first element in the tuple must match the names for races in players.json file